src/platform/utilities/api/index.js

```diff
@@ -97,6 +99,13 @@ export function apiRequest(resource, optionalSettings = {}, success, error) {
         ? response.json()
         : Promise.resolve(response);
 
+      // Get CSRF Token from API header
+      const csrfToken = response.headers.get('X-CSRF-Token');
+
+      if (csrfToken && !csrfTokenStored) {
```

Comment:
You're going to need to compare their values too; not just check for existence:
`if (csrfToken && csrfToken != csrfTokenStored) {`
If someone has a token set, then logs in, their token will change.
But FE will still have the old token in `localStorage`.  It won't get updated (because it exists), and CSRF validations will fail.

Reply:
Now, that the API is no longer changing the token, it makes sense.